### DISCUSSION TITLE
Differing scan results

### DISCUSSION THREAD

**Discussion by wellenreiten54** [immauss/openvas#304] [Q&A]
I have scanned the same systems at intervals of several months. In the first scan, 222 results were found. Now, suddenly, there are only 12 left (all of the type Log). However, no measures have been taken on the systems in the meantime that could have closed vulnerabilities. Is there an explanation for this?

**Comment by gooseleggs** [comment]
In both scans, there with be an _Authenticated scan results_ log entry.  Have a look and compare that.  Maybe one was authenticated, and the new one wasn't?  To see if, you may need to reduce the filter level down to 30%

**Reply by wellenreiten54** [reply]
both scans were unauthenticated. In the second scan, unlike the first, no TLS certificates or ports are displayed. It gives the impression that not all checks were performed.

**Reply by wellenreiten54** [reply]
Another piece of evidence supporting my suspicion is the duration of the scan. In the first scan, it was 106 minutes, but in the second, it was only 12.

**Comment by wellenreiten54** [comment]
I've found the following in the log now. These entries exist for several "tests". What could be the problem?

main:WARNING:2024-10-23 07h46.41 utc:158105:7d3083b8-abe1-4144-83bf-0072fa15b570: There was a problem trying to load secpod_xampp_detect.nasl, a dependency of XAMPP < 7.2.29, 7.3 < 7.3.16, 7.4 < 7.4.4 Configuration Vulnerability. This may be due to a parse error, or it failed to find the dependency. Please check the path to the file.

**Reply by immauss** [reply]
Hmm .... This looks like it might be an issue with the feed. First step, make sure you have the latest feeds.

```
docker exec -it <container name> /scripts/sync.sh 
```
Check the logs and make sure it has completed the sync and give it another try. 

Either way, definitely a problem with Greenbone and not the container, so you might want to try over at the Greenbone forums too. 

https://forum.greenbone.net 

-Scott

**Reply by wellenreiten54** [reply]
Thank you for your response. The feeds were always up to date. I have now installed the Community Edition containers from GB for testing purposes. It is now running smoothly. However, I can live with continuing to work on this version. Thank you very much.
